eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -212,6 +212,17 @@ jobs:
 
       # ===== BEGIN LIVE TEST RUNS =====
 
+      - task: AzurePowerShell@5
+        inputs:
+          azureSubscription: ${{ parameters.CloudConfig.ServiceConnection }}
+          azurePowerShellVersion: LatestVersion
+          workingDirectory: $(PackagePath)
+          pwsh: true
+          ScriptType: InlineScript
+          Inline: |
+            Update-AzConfig -DisplayBreakingChangeWarning $false
```

Comment:
Is this only an issue in JS? I've not seen other folks hitting it yet.

Reply:
Some languagse may not be impacted, for example, .NET uses Xml and the parsing seems to work with the breaking change warnings https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/src/Credentials/AzurePowerShellCredential.cs#L261

Python only parses text lines so it works fine too https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/azure/identity/_credentials/azure_powershell.py#L176

Reply:
@KarishmaGhiya would know more from the identity feature crew

Reply:
We're currently investigating how each language is affected. There are two things - one is the token parsing logic and the other is updating the output of az powershell command to "SecureString". Latter is something all languages will need to do. JS needs to do both.